src/features/chart/redux/actions.ts

```diff
@@ -0,0 +1,37 @@
+import { UPDATE_CHART_RANGE, UPDATE_CHART_DATA } from "./constants";
+import { ActionWithPayload } from "../../../utils/actions";
```

Comment:
This'll be much cleaner when (if) you add Typescript paths. You can clean it up to look like:
```ts
import { ActionWithPayload } from 'utils/actions';
```

Reply:
You can also do this just by setting
```
"baseUrl": "src",
```
in tsconfig (i.e. no need to set up `paths` too). Then you can do imports from any dir under `src`. That's what they've done in the project I'm currently working on.

Reply:
Oh right, I thought there would be more setup required... I'll give that a go!